src/Pizza.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom"
+
+const Pizza = (props) => {
+    let history = useHistory()
+    const [form, setForm] = useState({
+        name: '',
+        sauce: 'red',
+        quantity: '1',
+
+    })
+    const handleCheckbox = (e) => {
+        setForm({
+            ...form,
+            [e.target.name]: e.target.checked
+        })
+    }
+    const handleForm = (e) => {
+        setForm({
+            ...form,
+            [e.target.name]: e.target.value
+        })
```

Comment:
You could also combine `handleForm` and `handleCheckbox` into one function like we do here https://lambdaschool.instructure.com/courses/281/pages/objective-1-control-forms-and-form-inputs-checkbox-dropdown-in-custom-forms?module_item_id=351811